layouts/partials/cards/skill.html

```diff
@@ -12,7 +12,13 @@
 
           <img class="card-img-xs" src="{{ $logoImage.RelPermalink }}" alt="{{ .name }}" />
         {{ end }}
+        
+        {{ if .icon }}
+        {{ $iconName := .icon }}
+        <h5 class="card-title"><span class="{{ $iconName }}"> {{ .name }}</span></h5>
+        {{ else }}
         <h5 class="card-title">{{ .name }}</h5>
+        {{ end }}
```

Comment:
I think it is better to have little duplication if it improves the code readability You can make the code blocks as below:

```html
{{ if .logo }}
  {{ $logoImage := resources.Get .logo }}
  {{/*  svg don't support "Fit" operation   */}}
  {{ if ne $logoImage.MediaType.SubType "svg" }}
    {{ $logoImage = $logoImage.Fit "24x24" }}
  {{ end }}

  <img class="card-img-xs" src="{{ $logoImage.RelPermalink }}" alt="{{ .name }}" />
  <h5 class="card-title">{{ .name }}</h5>
{{ else if .icon }}
  {{ $iconName := .icon }}
  <h5 class="card-title"><span class="{{ $iconName }}"> {{ .name }}</span></h5>
{{ else }}
  <h5 class="card-title">{{ .name }}</h5>
{{ end }}
```